gradle-jdks/src/test/groovy/com/palantir/gradle/jdks/JdksExtensionProjectSpec.groovy

```diff
@@ -56,9 +68,9 @@ final class JdksExtensionProjectSpec extends IntegrationSpec {
                 jdks.jdkFor(JavaLanguageVersion.of(11), project).get().jdkFor(os).jdkFor(arch).jdkVersion.get()
             }
             
-            println('jdkVersion macos aarch64: ' + jdkVersionFor(Os.MACOS, Arch.AARCH64))
```

Comment:
Was a little worried that `BaselineJavaVersions` had this concept of `Os` leak into it, but it's fine, didn't see that jdkVersionFor is just defined for this test